env.bzl

```diff
@@ -59,10 +87,9 @@ def _create_env_build_file(rctx, env_name):
     )
 
 def _conda_create_impl(rctx):
-    conda_label = Label("@{}//:{}/condabin/conda{}".format(rctx.attr.conda_repo, rctx.attr.conda_dir, CONDA_EXT_MAP[get_os(rctx)]))
-    executable = str(rctx.path(conda_label))
+    executable = _user_chosen_executable(rctx, rctx.attr.use_mamba)
     env_name = rctx.name
-    _create_environment(rctx, executable, env_name)
+    _create_environment(rctx, env_name)
```

Comment:
`_create_environment` should get the executable, so it's not bothering with resolving anything, just using whatever was passed.

Reply:
`_create_environment` uses up to 2 executables: `conda` and `mamba`. `conda` is used for creating an empty environment. In the case where mamba is selected, `mamba` is used to update the empty environment. Would you prefer both executables are passed in? Like this: `_create_environment(rctx, conda_executable, user_chosen_executable, env_name)`

Reply:
Can't `mamba` be also used for creating an empty environment?

Reply:
Yeah it looks like it. https://mamba.readthedocs.io/en/latest/user_guide/mamba.html

Reply:
Ok, I implemented it this way now.